### ISSUE TITLE
Question about the old 1.2

### ISSUE THREAD

**Issue by SzotsGabor** [ChizhovVadim/CounterGo#2]
Hi Vadim,

There has been uncertainty about Counter 1.2 being 32-bit or 64-bit engine. See here: http://www.talkchess.com/forum/viewtopic.php?t=67250

So I would like to ask you if it makes a strength difference running it on a 32-bit or 64-bit PC. If they are different, we will have to split the results on the CCRL lists.

Gabor Szots
CCRL testing group

**Comment by ChizhovVadim** [comment]
Hi Gabor.

Counter 1.2 requires .NET Framework to run.
When I developed Counter 1.2, I used 32 bit computer with 32 bit OS Windows
(probably Windows XP) and 32 bit Microsoft .NET Framework.
But the same binary file become 64-bit process when run on 64-bit OS and 64
bit .NET Framework installed (at present times it is almost always true).

Anyway Counter 1.2 has 12x10 board representation and 64 bit probably
should not give noticeable increase.

Best Regards,
Vadim Chizhov.

2018-05-03 16:02 GMT+03:00 SzotsGabor <notifications@github.com>:
